content/concept-amendments.md

```diff
@@ -179,33 +182,52 @@ An address with a SignerList can disable the master key even if a regular key is
 * `tefNOT_MULTI_SIGNING`
 * `tefBAD_AUTH_MASTER`
 
-<!--{# to be added in a later version:
+
 ## OwnerPaysFee
 
-| Amendment ID | Status |
-|--------------|--------|
-| TBD | Coming soon |
+| Amendment ID                                                     | Status    |
+|:-----------------------------------------------------------------|:----------|
+| 9178256A980A86CF3D70D0260A7DA6402AAFE43632FDBCB88037978404188871 | In development. |
+
+Fixes an inconsistency in the way [transfer fees](concept-transfer-fees.html) are calculated between [OfferCreate](reference-transaction-format.html#offercreate) and [Payment](reference-transaction-format.html#payment) transaction types. Without this amendment, the initial sender of a transaction pays the transfer fees for offers that are executed as part of payment processing, but the holder of the issuances pays the transfer fee if an offer is executed in offer placement. With OwnerPaysFee, the holder of the issuances always pays the transfer fee, regardless of whether the offer is executed as part of a Payment or an OfferCreate transaction. Offer processing outside of payments is unaffected.
```

Comment:
> Without this amendment, the initial sender of a transaction pays the transfer fees for offers that are executed as part of payment processing, but the holder of the issuances pays the transfer fee if an offer is executed in offer placement. With OwnerPaysFee, the holder of the issuances always pays the transfer fee, regardless of whether the offer is executed as part of a Payment or an OfferCreate transaction. Offer processing outside of payments is unaffected.

To make this a little easier to follow, maybe make these sentences more parallel. E.g. "Without OwnerPaysFee, the holder of the issuance..."